debugging/coreclr/dockerManager.ts

```diff
@@ -236,7 +236,7 @@ export class DefaultDockerManager implements DockerManager {
 
         await this.addToDebugContainers(containerId);
 
-        const debuggerPath = await this.debuggerClient.getDebugger(options.run.os, containerId);
+        const debuggerPath = this.osProvider.pathNormalize(options.run.os, await this.debuggerClient.getDebugger(options.run.os, containerId));
```

Comment:
Is this the right place to do that?

Reply:
I feel like the `DebuggerClient` should be responsible for returning a relative path to the debugger *in the container platform's format*, so I've moved it there in PR #798, which is based on your workaround.

Reply:
Ok, I will remove this from my branch.